packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated WDIO screenshots
+
```

Comment:
We can probably omit this entry since consumers don't need to know about WDIO screenshot updates.

Reply:
Removed doc entry - https://github.com/cerner/terra-framework/pull/1816/commits/3ab310ec3215037ba8931ef92010230d2ed77748